### ISSUE TITLE
PostgreSQL parser doesn't check if a function exists

### ISSUE THREAD

**Issue by maxhawkins** [kyleconroy/sqlc#537]
```sql
-- name: F :exec
SELECT doesntexist();
```

Compiles using the experimental Postgres parser, but it shouldn't since the function does not exist.

**Comment by kyleconroy** [comment]
Now that #635 has been merged, we're closer to fixing this. One blocker is functions defined by extensions. I need to update the generation script to support all built-in extensions.

**Comment by kyleconroy** [comment]
I've made a first pass at adding extension functions (#640). I need to add support for all extensions that ship with PostgreSQL, and then I think we can turn this on.

**Comment by kyleconroy** [comment]
I've added all the functions from popular PostgreSQL extensions. The last step is to add a configuration option to turn this on. The other possibility would be to turn this on by default and let people turn it off. Not sure which option we should pick.

**Comment by maxhawkins** [comment]
Is it possible to activate when CREATE EXTENSION is called?

**Comment by kyleconroy** [comment]
This will be released in v1.12.0 behind a the `strict_function_checks` configuration flag.

```
{
	"version": "1",
	"packages": [
		{
			"name": "querytest",
			"path": "go",
			"schema": "query.sql",
			"queries": "query.sql",
			"engine": "postgresql",
			"strict_function_checks": true
		}
	]
}
```
